Update README (#424)
Exawind/openturbine: README.md

# OpenTurbine

**OpenTurbine development began in 2023, is ongoing, and we aim for beta release in 2025**

OpenTurbine is an open-source flexible-multibody-dynamics simulation
code designed to meet the research needs of Wind Energy Technologies Office (WETO)
and the broader wind energy community for land-based and offshore wind turbines.
OpenTurbine provides high-fidelity, highly performant structural dynamics
models that can couple with low-fidelity aerodynamic/hydrodynamic models like those
in [OpenFAST](https://github.com/OpenFAST/openfast), as well as high-fidelity
computational fluid dynamics (CFD) models like those in the WETO and Office
of Science supported [ExaWind](https://github.com/Exawind) code suite.

The following describes the high-level development objectives conceived for OpenTurbine:
- OpenTurbine will follow modern software development best practices, 
including test-driven development (TDD), version control,
hierarchical automated testing, and continuous integration (CI) for a
robust development environment.
- The core data structures are memory efficient and enable vectorization
and parallelization at multiple levels.

> - Data structures are data-oriented to exploit methods for accelerated computing including
> high utilization of chip resources (e.g., single instruction multiple data i.e. SIMD),
> parallelization through GP-GPUs or other hardware, and support for memory-efficient
> architectures.

Reviewer: ```suggestion
- Data structures are data-oriented to exploit methods for accelerated computing including
high utilization of chip resources (e.g., single instruction multiple data (SIMD) instruction sets) and
parallelization using GP-GPUs.
```
